pkg/cloud/impl_registry.go

```diff
@@ -245,18 +337,18 @@ func makeLimiter(
 	}
 	s.rate.SetOnChange(sv, fn)
 	s.burst.SetOnChange(sv, fn)
-	fn(ctx)
+	fn(context.Background())
```

Comment:
why not remove the context arg all together from `fn`?

Reply:
`fn` is passed to `SetOnChange` and needs to adhere to its arg type.